addons/mail/models/ir_actions_server.py

```diff
@@ -321,8 +327,36 @@ def _is_recompute(self):
                     return True
         return False
 
+    def _run_action_log_note_multi(self, eval_context=None):
+        if (not self.env.context.get('active_ids') and not self.env.context.get('active_id')) or self._is_recompute():
+            return False
+        res_ids = self.env.context.get('active_ids', [self.env.context.get('active_id')])
+
+        # Clean context from default_type to avoid making attachment
+        # with wrong values in subsequent operations
+        cleaned_ctx = dict(self.env.context)
+        cleaned_ctx.pop('default_type', None)
+        cleaned_ctx.pop('default_parent_id', None)
+        cleaned_ctx['mail_post_autofollow_author_skip'] = True  # do not subscribe random people to records
+        cleaned_ctx['mail_post_autofollow'] = self.mail_post_autofollow
+
+        # Creating a virtual template record
+        template = self.env['mail.template'].new({
+            'model_id': self.env['ir.model'].search([('model', '=', self.model_name)], limit=1).id,
+            'body_html': self.log_note_note,
```

Comment:
I don't think creating a new mail template is the way to go.
My first analysis was to also have a mail_template id for server action in log_note mode